src/app/sh-http/sh-http.service.js

```
@@ -12,29 +12,26 @@ export default class ShHttpService {
     get(url) {
         return this.http
             .get(url, this._getOptions('Get', url))
-            .map(this._convertToJson);
+            .map(this._convertToJson)
+            .catch(this._errorHandling);
```

!fix
please, verify: seems like all catches suppress errors